flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/BigUserProgramJobSubmitITCase.java

```diff
@@ -43,40 +40,27 @@
 /**
  * Integration test that verifies that a user program with a big(ger) payload is successfully
  * submitted and run.
+ *
+ * <p>This test fails for non-flip6 cluster due to job submission payload being too large, see [FLINK-7285].
  */
-@Ignore("Fails on job submission payload being too large - [FLINK-7285]")
+@Category(Flip6.class)
 public class BigUserProgramJobSubmitITCase extends TestLogger {
 
 	// ------------------------------------------------------------------------
 	//  The mini cluster that is shared across tests
 	// ------------------------------------------------------------------------
 
-	private static final int DEFAULT_PARALLELISM = 1;
-
-	private static LocalFlinkMiniCluster cluster;
-
-	private static final Logger LOG = LoggerFactory.getLogger(BigUserProgramJobSubmitITCase.class);
+	@ClassRule
+	public static final MiniClusterResource MINI_CLUSTER_RESOURCE = new MiniClusterResource(
+		new MiniClusterResource.MiniClusterResourceConfiguration(
+			new Configuration(),
+			1,
+			1));
 
 	// ------------------------------------------------------------------------
 	//  Cluster setup & teardown
 	// ------------------------------------------------------------------------
 
-	@BeforeClass
-	public static void setup() throws Exception {
-		// make sure we do not use a singleActorSystem for the tests
-		// (therefore, we cannot simply inherit from StreamingMultipleProgramsTestBase)
```

Comment:
Doesn't the `MiniClusterResource` always start the cluster with a single actor system? I'm wondering why this test succeeds then.

Reply:
maybe it does succeed exactly because of that. If everything runs in the same actorSystem the messagesa ren't subject to the `akka.framesize` limit.